src/modules/job-manager/alloc.c

```diff
@@ -63,7 +63,7 @@ static void requeue_pending (struct alloc *alloc, struct job *job)
     assert (job->alloc_pending);
     assert (job->handle == NULL);
     if (!(job->handle = zlistx_insert (alloc->queue, job, fwd)))
-        flux_log_error (ctx->h, "%s: queue_insert", __FUNCTION__);
+        flux_log_error (ctx->h, "%s: zlistx_insert", __FUNCTION__);
```

Comment:
If we're fixing error messages here
* `xlistx_insert()` doesn't claim to set errno, so `flux_log_error()` is inappropriate
* how about something more descriptive like "failed to enqueue job for scheduling" and drop  `__FUNCTION__`